classes/model/answer/field.model.php

```diff
@@ -44,7 +44,7 @@ class Model_Answer_Field extends \Nos\Orm\Model
     );
 
     protected static $_observers = array(
-        'Orm\\Observer_Typing',
+        'Nos\Form\Observer_Typing',
```

Comment:
The old answer fields are not serialized and the original observer doesn't handle non serialized value. This is a quick fix, feel free to propose something better.